.github/workflows/update_package_properties.yml

```diff
@@ -0,0 +1,52 @@
+name: Update Package Properties
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PRJ_NAME: ${{ github.event.inputs.project_name }}
+  TAG_NAME: ${{ github.event.inputs.tag_name }}
+  PRJ_VER: ${{ github.event.inputs.project_version }}
+  FANCY: ${{ github.event.inputs.fancy }}
+  FANCY_VERSION_NO: ${{ github.event.inputs.fancy_version_no }}
+  MICROSOFT_EMAIL: ${{ github.event.inputs.microsoft_email }}
+  NAME: ${{ github.event.inputs.name }}
+
+on:
+
+  workflow_dispatch:
+    inputs:
+      project_name:
```

Comment:
Since you wish to merge these changes into `all-citus` branch, won't the project name always be Citus? Do you wish to apply similar changes in other `all-X` branches as well?

I suggest we keep different workflow definitions for each project, and hardcode such parameters where possible. 
(i) This way we will have one less thing to worry about on a release since we will not need to swap between different projects
(ii) This way we will not lose our ability to use different versions of tools repository for different projects, and thus we will not break package creation for the less frequently updated projects such as topn and hll.

Reply:
This is actually an example of what @JelteF  asked earlier

> What is the repo layout of that we want in the future? Still one branch per project? If so, then I think having most of these values be configurable is not very useful. They can be hardcoded in the workflow since each workflow will have a different template.

Reply:
Removed projet_name parameter and hardocoded project_name as "citus"